bundles/binding/org.openhab.binding.expire/.gitignore

```diff
@@ -0,0 +1,3 @@
+/.classpath
+/.project
+/.settings/
```

Comment:
I committed that file accidentally. I had a look to the other plugins - seems that people use to commit those eclipse files. If this is general project style, I just remove my gitignore. (personally I think, this files should not be committed for maven projects because it makes the project less portable between IDEs but I have impression that it might be almost impossible to develop without the auto-installed eclipse version).